### PR TITLE
feat: add strict mode for return string format query param only

### PR DESCRIPTION
**Pull request by fengmk2**
Fixes koajs/querystring-strict#1

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 var request = require('supertest')
 var koa = require('koa')
+var urllib = require('urllib')
```
Comment (dead-horse) [inline test/test.js]: just use `supertest`?

Comment (fengmk2) [inline test/test.js]: supertest test will convert the 'p=a&p=b' to 'p[0]=a&p[1]=b' ...

**Comment by dead-horse** [comment]
LGTM

**Comment by fengmk2** [comment]
1.1.0 release

**Pull request merged**